### ISSUE TITLE
[Accessibility]A11y_ Azure Data Studio_Create Alert_Color contrast: Ratio is failing on "Create Alert" screen for buttons.

### ISSUE THREAD

**Issue by ceatcstt** [microsoft/azuredatastudio#5677]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows 10

**Additional Details:**
Tool Name: Color contrast 
MAS reference: MAS1.4.3

**Repro Steps:**
1. Install the app Azure Data Studio
2. Connect to the Database -> Go to the SQL server agent tab -> Navigate to "Alerts tab" by pressing tab key.
3. Click on "New Alert" button.
4. Open "Accessibility Insights" App for Windows.
5. Navigate to "Contrast" in "Accessibility Insights" App for Windows.
6. On "Create Alert" screen -> In General Tab.
7. With the help of mouse click on "Ok" and "Cancel" button background colour (blue) by picking up colour picker for Color 2.
8. With the help of mouse click on "Ok" and "Cancel" button foreground text by picking up colour picker for Color 1.

**Actual:**
Color Contrast ratio is showing as "3.458:1".

**Expected:**
Color Contrast ratio should be shown as >=4.5:1

**User Impact:**
User with disabilities find it difficult to understand the extra information provided.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS 1.4.3-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment for Reference:**
